python/cudf_polars/cudf_polars/dsl/expressions/datetime.py

```diff
@@ -58,6 +58,7 @@ class Name(IntEnum):
         OrdinalDay = auto()
         Quarter = auto()
         ReplaceTimeZone = auto()
+        Replace = auto()
```

Comment:
Sorting would probably give the other order:
```suggestion
        Replace = auto()
        ReplaceTimeZone = auto()
```